docs/src/tutorial.md

```diff
@@ -51,6 +51,10 @@ Go to `Retriever.jl/src`. Run Julia.
 
 ## Tutorial
 
+The Retriever has both build-in and recipe scripts.
+The Retriever's build-in scripts as part of the installation.
+The Retriever recipes are 3rd party scripts(datasets) from data package owners.
```

Comment:
How about "Each recipe provides the information for how to install a dataset. Recipes are contributed by a variety of users including data owners, data users, and the Retriever maintainers."